db/migrate/20170307090147_add_time_limit_to_existing_programming_questions.rb

```diff
@@ -0,0 +1,14 @@
+class AddTimeLimitToExistingProgrammingQuestions < ActiveRecord::Migration
+  def change
+    add_lower_default_to_existing_programming_questions
+  end
+
+  def add_lower_default_to_existing_programming_questions
+    Course::Assessment::Question::Programming.where(time_limit: nil).
```

Comment:
`Course::Assessment::Question::Programming.where(time_limit: nil).update_all(:time_limit, 10)` is much faster, but since there're not so many questions, I think this is okay.